setup.cfg

```diff
@@ -14,16 +14,19 @@ zip_safe = False
 include_package_data = True
 install_requires =
     pydantic
-    ga4gh.vrsatile.pydantic >=0.0.11
+    ga4gh.vrsatile.pydantic ==0.0.11
     ga4gh.vrs[extras] >=0.7.2
     biocommons.seqrepo
-    gene-normalizer >=0.1.27
-    uta-tools >=0.1.0
+    gene-normalizer ==0.1.27
+    uta-tools >=0.1.2
 
 tests_require =
     pytest
     pytest-cov
     pytest-asyncio
 
+[options.package_data]
+fusor/examples = **
+
```

Comment:
@korikuzma this is how you do this, right? I couldn't remember if we have an example to check against somewhere

Reply:
@jsstevenson  [Here](https://github.com/ga4gh/vrs-python/blob/metaschema-update/setup.cfg#L74) is an example of how we use it in vrs-python and [here](https://github.com/cancervariants/variation-normalization/blob/main/setup.cfg#L45) is another example of how we use it in variation-normalization